src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -220,16 +229,7 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
             SelectVersion();
 
             OnPropertyChanged(nameof(Versions));
-            OnPropertyChanged(nameof(VersionsView)); // This can change the selected version if a version is added after SelectedVersion is set
-
-            // If the selected version changed, reset it to the correct one
-            if (_nugetProjects.Any() &&
-                IsProjectPackageReference &&
-                SelectedVersion != FirstDisplayedVersion)
```

Comment:
Deleted these lines because there are no longer needed, before Text was Two-Way and was affecting SelectedVersion sometimes.